### ISSUE TITLE
Fail to follow ProjectReference if the referenced project is not included in the open Solution.

### ISSUE THREAD

**Issue by cruz82** [OmniSharp/omnisharp-roslyn#762]
Given the following structure

> Solution A
>    -> Project X references Project Y using ProjectReference
>
>Solution B
>   -> Project Y

The omnisharp compilation fails to follow the project reference if the referenced project is not included in the currently open solution (Solution A) with the following message
```
[warn]: OmniSharp.MSBuild.MSBuildProjectSystem
        Unable to resolve project reference 'ProjectY.csproj' for 'ProjectX'.
```

Using the dotnet CLI (1.0.0-rc4-004771), there is no issue building the solution.

**Comment by DustinCampbell** [comment]
I was planning to look at this as part of https://github.com/OmniSharp/omnisharp-vscode/issues/963.

**Comment by DustinCampbell** [comment]
This should be fixed with #832
